README.md

```diff
@@ -99,22 +101,71 @@ then [docker-compose-no-pleuston.yml](docker-compose-no-pleuston.yml) will be us
 If you do:
 
 ```bash
-./start_ocean.sh --latest --local-parity-node
+./start_ocean.sh --latest --local-secret-store
 ```
 
-then [docker-compose-local-parity-node.yml](docker-compose-local-parity-node.yml) will be used. Read it to see what images it starts. Note that it _doesn't_ start Pleuston, and it _does_ start a Parity Secret Store.
+then [docker-compose-local-secret-store.yml](docker-compose-local-secret-store.yml) will be used. Note that it _doesn't_ start Pleuston, and it _does_ start a Parity Secret Store. The images that are started would be:
+
+- mongo:3.6
+- oceanprotocol/parity-ethereum:master (for parity client)
+- oceanprotocol/parity-ethereum:master (for secret store)
+- nginx:alpine (for secret store cluster)
+- oceanprotocol/keeper-contracts:latest
```

Comment:
I verified that this list of Docker images is correct, but I'm confused about why `oceanprotocol/keeper-contracts:...` is included. If the user is running a local parity node, isn't that where the keeper contracts should be running? That is, they don't need a local Ganache node.

Reply:
keeper-contracts is used to deploy the ocean contracts to the parity network. As you said, ganache is not running in this case (so keeper-contracts acts as a job instead of a service)

Reply:
Ah, okay, thanks for clarifying.